core/FrontController.php

```diff
@@ -466,6 +466,20 @@ protected function prepareDispatch($module, $action, $parameters)
             && ($module !== 'API' || ($action && $action !== 'index'))
         ) {
             Session::start();
+
```

Comment:
I was wondering where to put this code here. If we had a Router (refs #6403) it could be maybe related to this. For now I'm ok with having it in FrontController as it is probably not worth it to write an API for this yet.